packages/tasks/src/model-libraries.ts

```diff
@@ -715,6 +715,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path:"model_vae_fp16.pt"`,
 		snippets: snippets.threedtopia_xl,
 	},
+	cotracker: {
```

Comment:
Could you move this entry up in the mapping to keep it consistent with the alphabetical order. Thanks!